### ISSUE TITLE
u5-select: Incorrect screen reader speech output after the items popup is closed

### ISSUE THREAD

**Issue by ndeshev** [SAP/ui5-webcomponents#2806]
**Describe the bug**
If ui5-select is placed within another component, for example ui5-panel, after closing the popover with the available items the screen reader is announcing the content of the parent component of the ui5-select.

**To reproduce**
1. Go to https://codesandbox.io/s/ui5-webcomponents-forked-9n7f0?file=/index.html
2. Launch your screen reader, the issue was originally detected with JAWS
2. Click on the Select component to open the items available for selection.
3. Choose the first or the second available items. Seems like the issue in this example is not reproducing if the last one is selected.
4. When the Select is within another component, after the select is closed, the screen reader is announcing its texts and not the ui5-select ones.

**Expected behavior**
After the select is closed the screen reader should not announce its parent container's content but the ui5-select instead.

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-p could you take a look of this issue.
